.upstream-tests/test/cuda/atomics/atomic.ext/atomic_fetch_min.pass.cpp

```diff
@@ -60,14 +60,59 @@ struct TestFn {
   }
 };
 
+template <template<typename, typename> typename Selector, cuda::thread_scope ThreadScope>
+struct TestFn<int, Selector, ThreadScope> {
+  __host__ __device__
+  void operator()() const {
+    // Test lesser
+    {
+        typedef cuda::atomic<int> A;
+        Selector<A, constructor_initializer> sel;
+        A & t = *sel.construct();
+        t = int(5);
+        assert(t.fetch_min(4) == int(5));
+        assert(t.load() == int(4));
+    }
+    {
+        typedef cuda::atomic<int> A;
+        Selector<volatile A, constructor_initializer> sel;
+        volatile A & t = *sel.construct();
+        t = int(5);
+        assert(t.fetch_min(-1) == int(5));
```

Comment:
This is changing the values of the test, when we actually want to only add volatile. I would either keep them the same of change them consistently throughout the file.

Reply:
We did want to change the test in this case, but there is greater value in making a proper unsigned/signed split test so that we can guarantee that add/sub/max/min are behaving correctly.